src/policy_artifacthub.rs

```diff
@@ -509,6 +509,7 @@ mod tests {
             background_audit: true,
             context_aware_resources: HashSet::new(),
             execution_mode: Default::default(),
+            policy_type: Default::default(),
```

Comment:
I think we should be explicit and make it `PolcyType::Kubernetes`

Reply:
done in https://github.com/kubewarden/policy-evaluator/pull/368/commits/8086300c601b4b29b1d5cac15213381b17bb88eb